components/bluetooth/lib.rs

```diff
@@ -120,20 +120,20 @@ fn matches_filter(device: &BluetoothDevice, filter: &BluetoothScanfilter) -> boo
     if !filter.get_services().is_empty() {
         if let Ok(device_uuids) = device.get_uuids() {
             for service in filter.get_services() {
-                if device_uuids.iter().find(|x| x == &service).is_none() {
+                if device_uuids.iter().any(|x| &x == &service) {
```

Comment:
I think this reverses the meaning of this if-statement. In the previous version it was checking if there were no results from the find and in this version it is true if there are any.

Reply:
@mrobinson I have updated this, please have a look